Makefile

```diff
@@ -32,7 +32,7 @@ endif
 
 assemble: clean resolve-registry build-server build-web-ui
 	@echo "Building docker image: ${CONTAINER_REGISTRY}/${IMAGE_NAME}:${IMAGE_TAG}"
-	docker rmi ${CONTAINER_REGISTRY}/${IMAGE_NAME}:${IMAGE_TAG}
+	docker rmi ${CONTAINER_REGISTRY}/${IMAGE_NAME}:${IMAGE_TAG} 2>/dev/null ; true
```

Comment:
You can also just use docker rmi -f (FYI).

Reply:
I wonder if we actually need to call `docker rmi` at all.

Reply:
Just tested it:
First is the image from yesterday:
```
aapregistrydev.azurecr.io/aoc-tznamena/installer                                     latest               1df0836ede00   22 hours ago    410MB
```
And after changing one line of code and rebuilding the same image:
```
aapregistrydev.azurecr.io/aoc-tznamena/installer                                     latest               a156f066a2ea   5 seconds ago   410MB
```
So the same image got new SHA. Hence I think we can remove the line removing the existing image.

Reply:
Sounds good. Updating...